### ISSUE TITLE
Choosing plots

### ISSUE THREAD

**Issue by ccarouge** [coecms/ncimonitor#5]
Maybe that's just me but the way to choose which plots to output was very confusing to me. If I don't specify anything, I get everything. If I specify something, I get only what I specify. I had to check the code to know how to change the default values. And that's only then I discovered that if I specify something then the other plots are set to False instead of True. 
Would it be clearer to have everything True by default and have to specify --no-X to set plot X to False?

**Comment by ccarouge** [comment]
Or maybe there needs to be a clearer help message?

**Comment by aidanheerdegen** [comment]
Ok. Point taken. I'll clean up the help message, as I prefer to specify a specific plot. It also scales if more plots are added. 

I can add probably add a `--no-` option

**Comment by ccarouge** [comment]
I agree it's nice to only specify what we want and it scales better.
